controllers/authController.ts

```diff
@@ -0,0 +1,96 @@
+import { NextFunction, Request, Response } from "express";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcryptjs";
+import { v4 as uuidv4 } from "uuid";
+import dotenv from "dotenv";
+import { PrismaClient } from "@prisma/client";
+import CreateError from "../utils/error";
+dotenv.config();
+const prisma = new PrismaClient();
```

Comment:
you can separate the db connection into a different file as `db.ts` and you can import them throughout the app.